core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -214,10 +214,8 @@ internal class CacheFallbackOperations
                 {
                     RemoveEntriesWithMatchingId(environmentAliases, clientId, accountOrUserId, adalCache);
                 }
-                else
-                {
-                    RemoveEntriesWithMatchingName(environmentAliases, clientId, displayableId, adalCache);
-                }
+
+                RemoveEntriesWithMatchingName(environmentAliases, clientId, displayableId, adalCache);
```

Comment:
There are unit tests for this logic, if changes are made, there should be unit test changes ( a bit surprised if they do no fail)

Reply:
This is covered by UnifiedCache_Adalv3ToAdalV4ToMsal2MigrationIntegrationTest